components/grouper/.gitignore

```diff
@@ -0,0 +1,4 @@
+/.project
+/.settings
+/.classpath
+/target
```

Comment:
Can't we stop adding these in every single project folder, and use a proper global .gitignore instead?

```
.project
.settings/
.classpath/
target/
/.idea
*.iml
´´´
Please correct if wrong; I believe all but .project are folders. I also took the liberty to add IDEA project folder and module files (the .idea folder is only created at top level, that's why it's prefixed with /).